### ISSUE TITLE
Api documentation seems to have issues with docblock backticks

### ISSUE THREAD

**Issue by kevinquinnyo** [cakephp/docs#2644]
For instance, a snippet from: http://api.cakephp.org/3.0/class-Cake.Network.Request.html as it appears in my browser:

```
query ( string $name )
Provides a read accessor for <code>$this-&gt;query</code>. Allows you to use a syntax similar to <code>CakeSession</code> for reading URL query data.
```

I haven't looked into this at all, but it seems like the backticks in your "doc blocks" aren't being handled correctly in whatever you are using to generate Api documentation, or the cakephp api documentation isn't handling the `<code>` tags correctly.

For instance, here is the docblock for the above example:

```
    /**
     * Provides a read accessor for `$this->query`. Allows you
     * to use a syntax similar to `CakeSession` for reading URL query data.
     *
     * @param string $name Query string variable name
     * @return mixed The value being read
     */
    public function query($name)
```

I know this is a minor gripe, but I thought you might want to be aware if you aren't and since I don't know enough about how you're parsing your docblocks to generate documentation I can't suggest a fix.

**Comment by HavokInspiration** [comment]
Looks like the same problem as here : cakephp/cakephp-api-docs#25

**Comment by markstory** [comment]
Closed with cakephp/cakephp-api-docs@064566adff2e14ef84ae67a853b3291dbc3db311
